src/main/docs/paths/load-guardian-name-path.ts

```diff
@@ -0,0 +1,63 @@
+export const loadGuardianNamePath = {
+  get: {
+    tags: ['guardian'],
+    summary: 'Load guardian name',
+    description: '',
+    security: [{
+      bearerAuth: []
+    }],
+    responses: {
+      200: {
+        description: 'Success',
+        content: {
+          'application/json': {
+            schema: {
+              type: 'object',
+              properties: {
+                firstName: {
+                  type: 'string'
+                },
+                lastName: {
+                  type: 'string'
+                }
+              },
+              example: {
+                firstName: 'John',
+                lastName: 'Doe'
+              }
+            }
+          }
+        }
+      },
+      400: {
```

Comment:
I think adding multiple answers breaks in some browsers or operating systems, give it a test there in windows before leaving it like that. :((

Reply:
I did not understand what you meant